Vungle/src/main/java/com/mopub/mobileads/VungleAdapterConfiguration.java

```diff
@@ -124,4 +130,19 @@ private void applyVungleNetworkSettings(Map<String, String> configuration) {
         VungleNetworkSettings.setMinSpaceForAdLoad(minSpaceLoadAd);
         VungleNetworkSettings.setAndroidIdOptOut(isAndroidIdOpted);
     }
+
+    private void refreshBidderToken() {
```

Comment:
Due to the IO performance risk issue: I checked FB implementation:
https://github.com/mopub/mopub-android-mediation/blob/197450b30a700f5d9f4cb9001fc769accd897a7b/FacebookAudienceNetwork/src/main/java/com/mopub/mobileads/FacebookAdapterConfiguration.java#L102